src/simulation/m_rhs.fpp

```diff
@@ -1028,7 +1077,8 @@ contains
         end if
         call cpu_time(t_finish)
         if (t_step >= 4) then
-            time_avg = (abs(t_finish - t_start) + (t_step - 4)*time_avg)/(t_step - 3)
+            ncells = max((ix%end - ix%beg), 1)*max((iy%end - iy%beg), 1)*max((iz%end - iz%beg), 1)
```

Comment:
I actually already fixed this here: https://github.com/MFlowCode/MFC/blob/4c3e34def7726d004fe69f4fe35c5ef80eb2d0c8/src/simulation/m_start_up.fpp#L1155 

But since it's in a different module, this doesn't overwrite my fix and will give wrong answers. Please revert this change.

Reply:
My change is actually necessary to get grind times that make sense (using m_glb instead of local domain sizes), hence me re-doing the performance table.